desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -15,11 +15,82 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import json
+from pprint import pprint
+
+from desktop import conf
 from django.shortcuts import render
 from django.http import HttpResponse
-from desktop.lib.django_util import login_notrequired
+from desktop.lib.django_util import login_notrequired, JsonResponse
+from desktop.lib.exceptions_renderable import PopupException
+from django.utils.translation import ugettext as _
+from django.views.decorators.csrf import csrf_exempt
+
+from django.conf import settings
+from slack_sdk import WebClient
+
+LOG = logging.getLogger(__name__)
+
+SLACK_VERIFICATION_TOKEN = conf.SLACK.SLACK_VERIFICATION_TOKEN.get()
+SLACK_BOT_USER_TOKEN = conf.SLACK.SLACK_BOT_USER_TOKEN.get()
+
+
+slack_client = WebClient(token=SLACK_BOT_USER_TOKEN)
+appname = "hue_bot"
```

Comment:
Will it be good if we get() the 'appname' also from the ini files just like other slack tokens @romainr ? This can help in separating it from dev and prod workspace with a different app in each one of them.
Or if there's any other more efficient way?